### ISSUE TITLE
wp_footer for settings

### ISSUE THREAD

**Issue by samikeijonen** [grappler/fluidvids#3]
Just curious, why you decided to use `wp_footer` to add settings JS and not `wp_enqueue_script`?

**Comment by grappler** [comment]
Because with `wp_enqueue_script` I would need to load a extra file and the variables need to be set with PHP. Is there a way to do it with `wp_enqueue_script`?

**Comment by samikeijonen** [comment]
You can use `wp_localize_script` function to set variables.

http://codex.wordpress.org/Function_Reference/wp_localize_script

https://pippinsplugins.com/making-php-variables-available-to-your-js/

**Comment by grappler** [comment]
Ok, but then you are loading an extra file which increases page load time.

**Comment by samikeijonen** [comment]
Yes I know. I haven't actually measured how many milliseconds faster `wp_footer` is than using `wp_enqueue_script` in footer. Also I'm not sure are we allowed to use `wp_footer` in our themes. For plugins pretty much everything goes:)
